pandas/core/internals/managers.py

```diff
@@ -169,27 +169,17 @@ def set_axis(self, axis, new_labels):
     def rename_axis(self, mapper, axis, copy=True, level=None):
         """
         Rename one of axes.
-
-        Parameters
+         Parameters
```

Comment:
can you undo this change?

Reply:
Copy/paste snafu, thanks for pointing this out.  Will fix shortly.